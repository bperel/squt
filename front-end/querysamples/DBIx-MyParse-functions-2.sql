SELECT 'Microsoft' SOUNDS LIKE 'Linix'
