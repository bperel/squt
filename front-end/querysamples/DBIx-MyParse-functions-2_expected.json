{
   "Functions" : {
      "1" : {
         "to" : "a",
         "Constants" : {
            "Microsoft" : "Microsoft"
         },
         "name" : "soundex",
         "alias" : "1"
      },
      "a" : {
         "to" : "OUTPUT",
         "name" : "=",
         "alias" : "a"
      },
      "2" : {
         "to" : "a",
         "Constants" : {
            "Linix" : "Linix"
         },
         "name" : "soundex",
         "alias" : "2"
      }
   }
}
